### DISCUSSION TITLE
i am not able to run it on the locahost

### DISCUSSION THREAD

**Discussion by kademlokesh** [ArslanYM/StarterHive#200] [Q&A] [answered]
I installed the node packages by command : npm install
but the outcome is :
--->StarterHive\web>npm i

added 330 packages, and audited 331 packages in 8s

107 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

--->StarterHive\web>npm start
npm ERR! Missing script: "start"
npm ERR!
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\kadem\AppData\Local\npm-cache\_logs\2023-06-03T14_49_48_297Z-debug-0.log

I am not able to run on my localhost

**Comment by ArslanYM** [comment]
@kademlokesh Is it working?

**Reply by kademlokesh** [reply]
no i updated my discussion section please look at this

**Comment by ArslanYM** [comment, marked as answer]
instead of running `npm start` try `npm run dev`. I think this will resolve the issue. If this works , make sure to click `this answered my question`  in this discussion panel

**Comment by kademlokesh** [comment]
this answered my question. thank u very much Arslan YM

**Reply by ArslanYM** [reply]
Glad to help.
